src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -57,7 +59,7 @@
  *
  * @author Kohsuke Kawaguchi
  */
-public class GitHubPushTrigger extends Trigger<Job<?, ?>> implements GitHubTrigger {
+public class GitHubPushTrigger extends Trigger<Job<?, ?>> implements GitHubTrigger2 {
```

Comment:
can we just drop this Interface implementation? Do you know somebody who use it?